picard/util/imagelist.py

```diff
@@ -165,6 +165,7 @@ def _update_state(obj, state):
 # TODO: use functools.singledispatch when py3 is supported
 def _get_state(obj):
     from picard.album import Album
+
```

Comment:
Why does it add this blank line?

Reply:
Because it separates picard and picard.ui imports according to the rules (separate import groups). The change now is that isort also handles imports placed elsewhere in the file, not only at the top. Which is actually great for us, because we have quite a few conditional imports (e.g. list of imports only in a `if IS_WIN`  block).